lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -25,6 +25,8 @@
 
 #include <mayaUsdUtils/util.h>
 
+#include <iostream>
```

Comment:
just noticed I left out to clean up this iostream. Will have it remove in another PR.